text_extensions_for_pandas/io/test_conll.py

```diff
@@ -50,7 +50,7 @@ def test_iob_to_spans(self):
                 """\
                                           span ent_type
                 0           [61, 67): 'Alaska'      GPE
-                1      [73, 84): 'Santa Claus'   PERSON
+                1      [73, 84): 'Santa Claus'      ORG
```

Comment:
I suppose this is more accurate, haha!

Reply:
:) Updates to the `en_core_web_sm` model tend to be "two steps forward, one step back".